### DISCUSSION TITLE
jQuery.isFunction() & jQuery.isArray are deprecated

### DISCUSSION THREAD

**Discussion by apasho** [vakata/jstree#2502] [General]
I'm testing jsTree v3.3.11 with using with jQuery 3.5.1. and getting jQuery.isFunction() & jQuery.isArray are deprecated. Wordpress is ending backward compatibility for older versions of jQuery in the next release. Can these depreciated functions be addressed?

**Comment by vakata** [comment]
I believe they already are - there were a series or merged PRs removing deprecated jQuery method calls.
